src/lib/viewers/doc/_docBase.scss

```diff
@@ -400,6 +400,26 @@ $thumbnail-sidebar-width: 191px; // Extra pixel to account for sidebar border
         }
     }
 
+    &.bp-annotations-only-controls {
+        &.bp-annotations-create--drawing,
+        &.bp-annotations-create--highlight,
+        &.bp-annotations-create--region {
+            // Show exit button when annotation-only controls is active
+            .bp-AnnotationsControls-exitBtn {
+                display: block;
+            }
+
+            // Hide all ControlBarGroups except for last instance which contains AnnotationsControls
+            // and hide all direct children of the group that isn't AnnotationControls
+            .bp-ControlsBarGroup {
+                &:not(:last-child),
+                & > *:not(.bp-AnnotationsControls) {
```

Comment:
We generally want to avoid `*` selectors and complex `:not` selectors, but I'm coming up empty on alternative solutions that don't involve more changes to the other controls. I don't think we need the `&` here, though.

Reply:
Alternatively, we could go back to my previous implementation where we use new class names to be a bit more explicit

Reply:
Yeah, why did we move away from below? Was it not covering all the use cases?

```
.bp-ControlsBarGroup:not(.bp-ControlsBarGroup-annotations) {
    display: none;
}

Reply:
There was some styling that wasn't ideal and that I had to add additional definitions to cover. I tried to find a way to not have to do that